Sources/NetworkService/HTTPMethod.swift

```diff
@@ -0,0 +1,11 @@
+/// The HTTP request method.
+public enum HTTPMethod: String {
+    /// The "GET" request.
```

Comment:
This is a method, not a request.

```suggestion
    /// The "GET" method.
```

Reply:
Fixed